config/common.unoconfig

```diff
@@ -6,6 +6,3 @@ if WIN32 {
     require "%HOME%/Library/Application Support/Fusetools/Fuse/Android/.sdkconfig" // For legacy 'fuse install android'
     require /usr/local/lib/node_modules/android-build-tools/.unoconfig // For 'npm install -g android-build-tools'
 }
-
-// User config
-require ~/.unoconfig
```

Comment:
Nit: I would kinda have expected this to be squashed in with 70f322d...

Reply:
Makes sense, I'll do that. :)